### PR TITLE
Eval 6 - CARLA Overhead Object Detection Test Data

### PR DESCRIPTION
**Pull request by yusong-tan**

### CODE REVIEW

**Comment by jprokos26** [comment]
I am getting slightly different values in the adversarial case for both the single & multimodal object detection. Currently reported values shown above my results - differences marked with *:

| Data | Defended | Attack            | Attack Parameters                  | Benign  mAP | Benign  Disappearance  Rate | Benign  Hallucination  per Image | Benign  Misclassification  Rate | Benign  True Positive  Rate | Adversarial  mAP | Adversarial  Disappearance  Rate | Adversarial Hallucination  per Image | Adversarial Misclassification  Rate | Adversarial True Positive  Rate | Test Size |
|------|----------|-------------------|------------------------------------|-------------|-----------------------------|----------------------------------|---------------------------------|-----------------------------|------------------|----------------------------------|--------------------------------------|-------------------------------------|---------------------------------|-----------|
| Test | no       | Adversarial Patch | learning_rate=0.003, max_iter=1000 | 0.60        | 0.42                        | 3.6                              | 0.03                            | 0.55                        |  0.04            | 0.81                             | 54.1                                 | 0.0                                 | 0.19                            | 15        |
| Test* | no       | Adversarial Patch | learning_rate=0.003, max_iter=1000 | 0.60        | 0.42                        | 3.6                              | 0.03                            | 0.55                        |  0.055*            | 0.82*                             | 55.3*                                 | 0.0                                 | 0.185*                            | 15        |

| Data | Defended | Attack            | Attack Parameters                                                                       | Benign  mAP | Benign  Disappearance  Rate | Benign  Hallucination  per Image | Benign  Misclassification  Rate | Benign  True Positive  Rate | Adversarial  mAP | Adversarial  Disappearance  Rate | Adversarial Hallucination  per Image | Adversarial Misclassification  Rate | Adversarial True Positive  Rate | Test Size |
|------|----------|-------------------|-----------------------------------------------------------------------------------------|-------------|-----------------------------|----------------------------------|---------------------------------|-----------------------------|------------------|----------------------------------|--------------------------------------|-------------------------------------|---------------------------------|-----------|
| Test | no       | Adversarial Patch | depth_delta_meters=0.03, learning_rate=0.003, learning_rate_depth=0.0001, max_iter=1000 | 0.58        | 0.39                        | 0.8                              | 0.03                            | 0.58                        | 0.19             | 0.72                             | 15.8                                 | 0.01                                | 0.23                            | 15        |
| Test* | no       | Adversarial Patch | depth_delta_meters=0.03, learning_rate=0.003, learning_rate_depth=0.0001, max_iter=1000 | 0.58        | 0.39                        | 0.8                              | 0.03                            | 0.58                        | 0.215*             | 0.715                             | 14.0*                                | 0.00774                                | 0.277*                            | 15        |

As a confirmation, this dataset should be using version 1.0.0 as shown below correct?
`downloaded /armory/datasets/cache/carla_over_od_test_cached_1.0.0.tar.gz from https://armory-public-data.s3.us-east-2.amazonaws.com/carla/carla_over_od_test_cached_1.0.0.tar.gz`

**Comment by yusong-tan** [comment]
Given that the attack uses random initialization, some slight discrepancy in results are expected. Results between @swsuggs and @jprokos26 look to be within reasonable error margin due to randomization.

File: docs/baseline_results/carla_od_results.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Single Modality (RGB) Object Detection
...
 | Dev  | no       | Robust DPatch     | learning_rate=0.002, max_iter=2000 | 0.64        | 0.36                        | 1.9                              | 0.02                            | 0.62                        |  0.42            | 0.43                             | 12.0                                 | 0.009                               | 0.56                            | 20        |
 | Dev  | yes      | Robust DPatch     | learning_rate=0.002, max_iter=2000 | 0.51        | 0.44                        | 3.1                              | 0.02                            | 0.54                        |  0.41            | 0.52                             | 9.8                                  | 0.01                                | 0.47                            | 20        |
-
+| Test | no       | Adversarial Patch | learning_rate=0.003, max_iter=1000 | 0.60        | 0.42                        | 3.6                              | 0.03                            | 0.55                        |  0.04            | 0.81                             | 54.1                                 | 0.0                                 | 0.19                            | 15        |
```
Comment (jprokos26) [inline docs/baseline_results/carla_od_results.md:42]: Very minor fixup but there should be mention of how Test results were obtained as done [here](https://github.com/twosixlabs/armory/pull/1851/files#diff-d9c10366f3697cde026d6e1a205abb6f35a5202f88bcb7c33705f51a256702cbR33). Results shown in https://github.com/twosixlabs/armory/pull/1851#issuecomment-1372603756 were generated using 0.16.1.post108+g72512cdc.d20230105 (72512cdc472cdb8f7bf86aaf8e93045788550848
)

**Review by jprokos26 — APPROVED** [review]
Approved in lieu of https://github.com/twosixlabs/armory/pull/1851#issuecomment-1372610625 once comment has been addressed.

**Pull request merged**